cmd/embedded-cluster/install.go

```diff
@@ -280,7 +280,7 @@ var installCommand = &cli.Command{
 		},
 		&cli.StringSliceFlag{
 			Name:  "disable-addon",
-			Usage: "Disable addon during install/upgrade",
+			Usage: "Disable addon during install",
```

Comment:
But I'm all for just getting rid of this one
```suggestionx
			Usage: "Disable add-on during install",
```

Reply:
This will be done in a separate PR.